doc/rados/operations/operating.rst

```diff
@@ -352,7 +352,29 @@ To stop a Ceph daemon on another node, use the following syntax::
 	sudo service ceph -a stop osd.0
 
 
+Stopping an Entire Cluster
+==========================
 
+To stop an entire ceph cluster (for example, if you must physically move
+hosts in a data center) first set the noout flag to prevent OSDs from being
+removed::
+	
+	ceph osd set noout
+
+Then shutdown all the services in your cluster in the manner appropriate for your platform.
+
```

Comment:
Sorry I am not an expert on this, but it really confuse me, in some of the doc, we explicitly use "service ceph -a stop osd" , but here you want to prevent the specificity. So what's the general rules we want to maintain?